MAINTAINERS.md

```diff
@@ -22,3 +22,5 @@ The maintainers [watch](https://github.com/Particular/ServiceInsight/watchers) t
 - After a pull request is merged, the milestone representing the next version should be added to:
   - The issue(s) which the PR is addressing, *or*
   - The PR itself, if it is not addressing any pre-existing issue.
+- If merging using the Github UI, the pull requests must not be merged using the "Squash and Merge" option as the Squash and Merge option creates a ff mmerge.
+- Each individual commit within a pull request must relate to a complete unit of work. If any "fix up" commits are made, they must be squashed into the pull request before merging.
```

Comment:
The fixup commits aren't really squashed "into the pull request". I would simplify it to:

> they must be squashed before merging.

We may also want to call out that the workflow should like this:

- Reviewer: Request changes or comment.
- Submitter: Push fixup.
- (Repeat)
- Reviewer: Request squash.
- Submitter: Squash and force push.
- Reviewer: Approve.
- (Any maintainer): Merge.